tests/compile-all/build.sh

```diff
@@ -132,14 +135,23 @@ do
 
             # Build the goal
             $LOG_INFO "make -C \"$example_dir\" -j TARGET=$platform BOARD=$board $GOAL"
-            if make -C "$example_dir" -j TARGET=$platform BOARD=$board $GOAL 2>&1 >build.log
+            if make -C "$example_dir" -j TARGET=$platform BOARD=$board $GOAL >build.log 2>&1
             then
                 $LOG_INFO "..done"
                 $CAT_DEBUG build.log
-                NUM_SUCCESS=$(($NUM_SUCCESS + 1))
+                if [[ `grep Skipping build.log` ]]
```

Comment:
I cannot immediately think where this is getting set. Is this for the cooja platform skipping only?